CHANGELOG.md

```diff
@@ -1,6 +1,79 @@
+# 1.7.0-rc01 (September 2024)
+
+_Changes since 1.7.0-beta02_
+
+## Highlights
+
+### Web
+
+- [`skiko.js` is redundant in case of K/Wasm Compose for Web applications and it can be removed from index.html files to not load redundant files](https://github.com/JetBrains/compose-multiplatform/pull/5134). `skiko.js` will be removed from the k/wasm distribution in the future releases. `skiko.js` is still needed in case of K/JS Compose for Web apps
```

Comment:
```suggestion
- [`skiko.js` is redundant in case of K/Wasm Compose Multiplatform for web applications and it can be removed from index.html files to not load redundant files](https://github.com/JetBrains/compose-multiplatform/pull/5134). `skiko.js` will be removed from the k/wasm distribution in the future releases. `skiko.js` is still needed in case of K/JS Compose for Web apps
```

For uniformity with other communications.

Reply:
Done